website/content/docs/configuration/seal/seal-ha.mdx

```diff
@@ -8,6 +8,7 @@ description: |-
 # Seal High Availability
 
 <EnterpriseAlert product="vault" />
+@include 'alerts/beta.mdx'
```

Comment:
I think you need a newline between these two.

<img width="724" alt="Screenshot 2024-03-27 at 2 57 24 PM" src="https://github.com/hashicorp/vault/assets/3989899/3fa93726-d566-4cee-b4df-d7fb2e11cf1a">

Reply:
yeah, just fixed it